python/player.py

```diff
@@ -13,13 +25,13 @@ def __init__(self, inventory: Inventory, stats: Stats) -> None:
         self.inventory = inventory
 
     def calculate_damage(self, other: Target) -> Damage:
-        base_damage = self.__get_base_damage()
-        damage_modifier = self.__get_damage_modifier()
+        base_damage = self._get_base_damage()
```

Comment:
I did not even see that. @emilybache was there a reason?

Reply:
I am not aware of double underscore in python. double leading underscores triggers name mangling in Python. While it makes them inaccessible aka private I have never seen it used in python. It surprises most devs

Reply:
I don't agree that it surprises most devs, I would say it is common knowledge in the python community. It is part of the official documentation and tutorial https://docs.python.org/3/tutorial/classes.html#private-variables. I don't think you can argue that it's not expected by a professional python programmer to know whats in the official tutorial, or that if they don't, they should learn it when they stumble upon one it.

I also find it a lazy argument, since if it was true that underscores are confusing, then why have them at all? then the members should just be `get_base_damage` which I don't think help anyone trying to learn proper Python programming. The name mangling argument is only proving it clearer, since it truly is private if you don't know how to call it from outside of the class, and that is the point!